site/config.toml

```diff
@@ -95,6 +95,8 @@ weight = 1
 copyright_k8s = "The Kubernetes Authors"
 copyright_linux =  "Copyright © 2020 The Linux Foundation ®."
 
+latest_version = "v4.5.4"
```

Comment:
I like this, but I'm worried it will immediately get stale. Options include:
- Change the name to something like "kustomize_example_version" to not imply it is necessarily the most current
- Update the release process to include bumping this. We already have a step [here](https://github.com/kubernetes-sigs/kustomize/tree/master/releasing#update-example-test-target) where we update a hardcoded string to make the example tests run against latest. If we were embedding those examples in the site (someday 🤞 ), it could make a lot of sense to sync that version with the one we advertise on the site. We could add a makefile target that does both, and update the release instructions to use that instead.

Wdyt?

Reply:
I like the idea of option 2 where we bump it during every release, but I don't think the site is ready for that yet. In the future when the site is complete we can come back and revisit this.

For now I'll change the name of the value to `kustomize_example_version` which is a good call! Less confusion if it strays from the official latest release.